care/facility/api/serializers/patient.py

```diff
@@ -188,11 +188,25 @@ def create(self, validated_data):
 class PatientSearchSerializer(serializers.ModelSerializer):
     gender = ChoiceField(choices=GENDER_CHOICES)
     phone_number = PhoneNumberIsPossibleField()
+    facility_id = serializers.SerializerMethodField(read_only=True)
 
     class Meta:
         model = PatientSearch
         fields = "__all__"
 
+    def get_facility_id(self, obj):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5398997)